CMakeLists.txt

```diff
@@ -28,7 +28,20 @@ option(RE2C_BUILD_LIBS "Build libraries" OFF)
 option(RE2C_BUILD_RE2GO "Build re2go executable (an alias for `re2c --lang go`)" ON)
 
 # checks for programs
-find_package(BISON)
+if(UNIX)
+    find_program(
+      BISON_EXECUTABLE
+      NAMES bison
+      PATHS /usr /usr/local
+      PATH_SUFFIXES bin)
+elseif(WIN32)
+    find_program(
+      BISON_EXECUTABLE
+      NAMES bison.exe win_bison.exe
+      PATHS C:/
+      PATH_SUFFIXES "")
+endif()
```

Comment:
Can you change this to:
```cmake
if(WIN32)
    find_program(BISON_EXECUTABLE
        NAMES bison.exe win_bison.exe
        PATHS C:/
        PATH_SUFFIXES "")
else()
    find_package(BISON)
endif()
```
Ideally I'd hope that [FindBISON](https://cmake.org/cmake/help/latest/module/FindBISON.html) would work for Windows, but maybe the paths are specific to this setup.

Also, note that bison is only needed to rebuild parser, it shouldn't be needed to build first time (pre-generated bootstrap/src/parse/parser.cc is used for that).

Reply:
I tried various find_package configurations on Windows - no luck :/ Also I noticed all we use below is BISON_EXECUTABLE. 
We just not use the power of find_package. Thus, I replaced find_package by find_program and set BISON_EXECUTABLE explicitly. However, your change looks reasonable and works well. Thank you.

Reply:
It is true that we do not need the full power of `find_package`. I just thought that `find_package` is the default CMake way and it is expected to encapsulate platform-specific details, but maybe I misunderstand its purpose.